backend/api/test_serializers.py

```diff
@@ -241,95 +248,234 @@ def test_missing_certifying_auditor(self):
 
     def test_auditor_list_not_all_valid_emails(self):
         auditor_not_all_valid_emails = {
             "certifying_auditee_contact_email": "a@a.com",
-            "certifying_auditor_contact_email": "b@b.com",
-            "auditor_contacts_email": ["c@c.com", "this is not an email", "d@d.com"],
-            "auditee_contacts_email": ["e@e.com", "f@f.com"],
+            "certifying_auditee_contact_fullname": "Fuller A. Namesmith",
+            "certifying_auditor_contact_email": "a@b.com",
+            "certifying_auditor_contact_fullname": "Fuller B. Namesmith",
+            "auditor_contacts_email": ["e@e.com", "this is not an email", "f@f.com"],
+            "auditor_contacts_fullname": [
+                "Fuller C. Namesmith",
+                "Fuller CC. Namesmith",
+                "Fuller CCC. Namesmith",
+            ],
+            "auditee_contacts_email": ["a@d.com", "a@e.com"],
+            "auditee_contacts_fullname": [
+                "Fuller D. Namesmith",
+                "Fuller DD. Namesmith",
+            ],
         }
-        self.assertFalse(
-            AccessAndSubmissionSerializer(data=auditor_not_all_valid_emails).is_valid()
+
+        serializer = AccessAndSubmissionSerializer(data=auditor_not_all_valid_emails)
+        self.assertFalse(serializer.is_valid())
+
+        auditor_contacts_email_errors = serializer.errors.get(
+            "auditor_contacts_email", None
         )
+        self.assertIsNotNone(auditor_contacts_email_errors)
 
     def test_auditee_list_not_all_valid_emails(self):
         auditee_not_all_valid_emails = {
             "certifying_auditee_contact_email": "a@a.com",
-            "certifying_auditor_contact_email": "b@b.com",
-            "auditor_contacts_email": ["c@c.com", "d@d.com"],
+            "certifying_auditee_contact_fullname": "Fuller A. Namesmith",
+            "certifying_auditor_contact_email": "a@b.com",
+            "certifying_auditor_contact_fullname": "Fuller B. Namesmith",
             "auditee_contacts_email": ["e@e.com", "this is not an email", "f@f.com"],
+            "auditee_contacts_fullname": [
+                "Fuller C. Namesmith",
+                "Fuller CC. Namesmith",
+                "Fuller CCC. Namesmith",
+            ],
+            "auditor_contacts_email": ["a@d.com", "a@e.com"],
+            "auditor_contacts_fullname": [
+                "Fuller D. Namesmith",
+                "Fuller DD. Namesmith",
+            ],
         }
-        self.assertFalse(
-            AccessAndSubmissionSerializer(data=auditee_not_all_valid_emails).is_valid()
+
+        serializer = AccessAndSubmissionSerializer(data=auditee_not_all_valid_emails)
+        self.assertFalse(serializer.is_valid())
+
+        auditee_contacts_email_errors = serializer.errors.get(
+            "auditee_contacts_email", None
         )
+        self.assertIsNotNone(auditee_contacts_email_errors)
+
+    def test_certifiers_same_email(self):
```

Comment:
many of the tests in this class were not actually testing what they claimed to be -- they asserted that the serializer validation failed, but in reality it was failing due to missing fields in the test data and not for the reasons specified in the test. They've been updated to assert that the validation failed for the expected reason(s)